test/input-format.js

```diff
@@ -0,0 +1,34 @@
+const test = require('brittle')
+const ffmpeg = require('..')
+
+test('input format should wrap an array buffer', (t) => {
+  let inputFormat
+
+  if (Bare.platform == 'darwin') {
+    inputFormat = new ffmpeg.InputFormat('avfoundation')
+  } else if (Bare.platform == 'linux') {
+    inputFormat = new ffmpeg.InputFormat('v4l2')
+  } else if (Bare.platform == 'win32') {
+    inputFormat = new ffmpeg.InputFormat('dshow')
+  }
+
+  t.ok(inputFormat instanceof ffmpeg.InputFormat)
```

Comment:
```suggestion
  t.ok(inputFormat)
```

Should be sufficient.